migrations/versions/e46b30dd3ff5_automatic_collections.py

```diff
@@ -0,0 +1,76 @@
+"""automatic_collections
+
+Revision ID: e46b30dd3ff5
+Revises: 9e33fa16ba82
+Create Date: 2023-02-06 11:44:29.000880
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e46b30dd3ff5'
+down_revision = '9e33fa16ba82'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "automatic_collection",
+        sa.Column(
+            'collection_id',
+            sa.Integer,
+            sa.ForeignKey('collections.id', ondelete='CASCADE'),
+            primary_key=True,
+        ),
+        sa.Column(
+            'keyword_query',
+            sa.String,
+        ),
+        sa.Column(
+            'author_query',
+            sa.String,
+        ),
+        sa.Column(
+            'title_query',
+            sa.String,
+        ),
+        sa.Column(
+            'subject_query',
+            sa.String,
+        ),
+        sa.Column(
+            'sort_field',
+            sa.String,
+            sa.CheckConstraint(r"sort_field is NULL OR sort_field IN ('title', 'author', 'date')"),
+        ),
+        sa.Column(
+            'sort_direction',
+            sa.String,
+            sa.CheckConstraint(r"sort_direction is NULL OR sort_direction IN ('ASC', 'DESC')"),
+        ),
```

Comment:
Should these be required with a default value of something like `title ASC` (or even `uuid ASC`)? Without a sort value we might see ordering change in these queries? Providing default behavior might ensure more consistency for patrons looking at these collections.

Reply:
Ah yea, maybe uuid would make sense, as it doesn't implement a semantically meaningful sort, but still keeps the sort stable